set.mm

```diff
@@ -645588,6 +645803,128 @@ with respect to a linear (1-dimensional) relation.  (Contributed by BJ
+      WRALJUCLUHZUDZKUEZXBXCFUDZXBCUFZUGWRBEUIMXBJUEZXDWOXBXFJWQXGXCWNKUCXBJUJZ
+      UKXGULZXGXEWPXBJCXGXCWNFXHUMXIUQUNUBAWOJWQBUIUBAWPJCUOUPURUSWSXAUEAWOUCJW
+      QUTVAAWOWNWTKDAWOVBAWOVCZVDZWTWPWNGVEUDZUFZDXKWPIVFUDZGCXLUCHVGUFZUCGVFUD
+      ZUIJIOXNVHZXPVHZXKUCHVGUOAGVIVLZWNXPVLZXJWNKVJZWPXPVLTAXOXPUCJAXOUIVLJXNV
+      LZXOXPJVKAUCHVGUOAJXNIVMUDWBZAJBXNYCVNUBNVOVPZXNGIXOXPUIJOXRXQVQVRAHWCVLU
+      CXOVLZUAHVSWDZVTZWNKWEZXPGFWNKXRQSWAWFAYBXJYDWGAYEXJYFWGPXLVHZWHAXSXTXJYA
+      XMDUETYGYHXPGXLDFWNKXRQYIRSWIWFWLWJWK $.
+  $}
+
+  ${
+    $d B x y $.  $d X l x y $.  $d W l x y $.  $d .x. l x y $.  $d S l $.
+    $d I l x y $.  $d ph x y $.  $d .0. x y $.  $d B b $.  $d X b $.
+    $d .0. b $.  $d .x. b $.  $d I b $.  $d ph b $.
+    prjspner01.e $e |- .~ = { <. x , y >. |
+      ( ( x e. B /\ y e. B ) /\ E. l e. S x = ( l .x. y ) ) } $.
+    prjspner01.f $e |- F = ( b e. B |->
+      if ( ( b ` 0 ) = .0. , b , ( ( I ` ( b ` 0 ) ) .x. b ) ) ) $.
+    prjspner01.b $e |- B = ( ( Base ` W ) \ { ( 0g ` W ) } ) $.
+    prjspner01.w $e |- W = ( K freeLMod ( 0 ... N ) ) $.
+    prjspner01.t $e |- .x. = ( .s ` W ) $.
+    prjspner01.s $e |- S = ( Base ` K ) $.
+    prjspner01.0 $e |- .0. = ( 0g ` K ) $.
+    prjspner01.i $e |- I = ( invr ` K ) $.
+    prjspner01.k $e |- ( ph -> K e. DivRing ) $.
+    prjspner01.n $e |- ( ph -> N e. NN0 ) $.
+    prjspner01.x $e |- ( ph -> X e. B ) $.
+    $( Any vector is equivalent to some vector whose value on the 0th axis is
+       either ` .0. ` or ` .1. ` : equivalence part.  (Contributed by SN,
+       13-Aug-2023.) $)
+    prjspner01 $p |- ( ph -> X .~ ( F ` X ) ) $=
+      ( cc0 cfv wceq co cif wbr wif prjspner erref adantr wn wa wer cdr wne cfz
+      wcel cvv cbs wf ovexd c0g csn cdif eleqtrdi eldifad eqid frlmbasf syl2anc
+      cn0 0elfz syl ffvelrnd neqne drnginvrcl syl2an3an prjspnvs ersym ifpimpda
+      drnginvrn0 brif2 sylibr cv fveq1 eqeq1d id fveq2d oveq12d ifbieq12d ifexd
+      fvmptd3 breqtrrd ) AMUHMUIZNUJZMWTIUIZMGUKZULZMHUIEAXAMMEUMZMXCEUMZUNMXDE
+      UMAXAXEXFAXEXAAMEDABCDEFGJKLPQTSUBUAUEUOZUGUPUQAXAURZUSZXCMEDADEUTXHXGUQX
+      IBCDXBEFGJKLMNPQTSUBUAUCAJVAVDZXHUEUQAMDVDXHUGUQAXJWTFVDZXHWTNVBZXBFVDUEA
+      UHKVCUKZFUHMAXMVEVDMLVFUIZVDXMFMVGAUHKVCVHAMXNLVIUIVJZAMDXNXOVKUGSVLVMXNJ
+      LXMFVEMTUBXNVNVOVPAKVQVDUHXMVDUFKVRVSVTZWTNWAZFJIWTNUBUCUDWBWCAXJXKXHXLXB
+      NVBUEXPXQFJIWTNUBUCUDWGWCWDWEWFXAMXCMEWHWIAOMUHOWJZUIZNUJZXRXSIUIZXRGUKZU
+      LXDDHVERXRMUJZXTXAXRYBMXCYCXSWTNUHXRMWKZWLYCWMZYCYAXBXRMGYCXSWTIYDWNYEWOW
+      PUGAXAMXCDVEUGAXBMGVHWQWRWS $.
+
+    $d Y l m x y $.  $d Y b $.  $d B m $.  $d I m $.  $d X m $.  $d .x. m $.
+    $d ph m $.  $d S m x y $.
+    prjspner1.y $e |- ( ph -> Y e. B ) $.
+    prjspner1.1 $e |- ( ph -> ( X ` 0 ) =/= .0. ) $.
+    prjspner1.2 $e |- ( ph -> ( Y ` 0 ) =/= .0. ) $.
+    $( Vectors whose first coordinate is nonzero are equivalent iff they
+       intersect the (n-1)-th dimensional space (offset 1 from the origin) at
```

Comment:
Two vectors whose zeroth coordinate is nonzero are equivalent if and only if they have the same representative in the (n-1)-dimensional affine subspace `{ x_0 = 1 }`.